src/components/daydream-controls.js

```diff
@@ -18,14 +16,16 @@ var GAMEPAD_ID_PREFIX = 'Daydream Controller';
  */
 module.exports.Component = registerComponent('daydream-controls', {
   schema: {
-    hand: {default: DEFAULT_HANDEDNESS}, // This informs the degenerate arm model.
+    hand: {default: ''}, // This informs the degenerate arm model.
```

Comment:
so we have had some discussion here.  if you remove the hand property that is used to filter so it only matches a particular hand,  then it is not possible to specify which handedness model should be used correctly.  i thought we had generally agreed to keep the filter property, but to default it to a value that means don't filter and simply accept whatever is there ("either", or blank).